Sources/Network/Rest/RequestAuthorizer.swift

```diff
@@ -10,4 +10,5 @@ import Foundation
 
 public protocol RequestAuthorizer {
     func authorize(request: URLRequest, completion: @escaping (Result<URLRequest, Error>) -> Void)
+    func refreshAuthorization(completion: @escaping (Result<Void, Error>) -> Void)
```

Comment:
```suggestion
    func refresh(completion: @escaping (Result<Void, Error>) -> Void)
```

Reply:
Wouldn't `refresh` be too common to use?

Reply:
Maybe. But my thinking is that Authorizer can refresh only authorization parameters